spec/factories/user.rb

```diff
@@ -0,0 +1,12 @@
+FactoryBot.define do
+  factory :user do
+    first_name { 'John' }
+    last_name { 'Doe' }
```

Comment:
Lets use the gem Ffaker to generate random strings
https://github.com/ffaker/ffaker